fluidBuild.config.cjs

```diff
@@ -119,20 +119,28 @@ module.exports = {
 			"tools/markdown-magic/test",
 			"tools/telemetry-generator/package-lock.json", // Workaround to allow version 2 while we move it to pnpm
 		],
+		// Exclusion per handler
+		handlerExclusions: {
+			"npm-package-json-script-clean": [
```

Comment:
Add a comment with the info from the pr description. That is, the part that explains why these packages don't have clean scripts.